README.md

```diff
@@ -698,6 +698,7 @@ Table of Contents
   * [hasura.io](https://hasura.io/) — Platform to build and deploy app backends fast, free for single node cluster.
   * [iron.io](https://www.iron.io/) — Async task processing (like AWS Lambda) with free tier and 1-month free trial
   * [netlicensing.io](https://netlicensing.io) - A cost-effective and integrated Licensing-as-a-Service (LaaS) solution for your software on any platform from Desktop to IoT and SaaS. Basic Plan for *FREE* while you are a student.
+  * [nhost.io](https://nhost.io) - Build apps fast without managing infrastructure - Get a complete backend in seconds with PostgreSQL, GraphQL, Hasura, Authentication, Storage and Serverless Functions.
```

Comment:
looks to me this is just a 14 day trial and not a free tier?

Reply:
Yea, we should wait to merge this for another 1-2 months before we get our free tier in place. I've updated the title to "[WIP]".